docs/v-slice/components/NoteStyles.md

```diff
@@ -0,0 +1,70 @@
+# Notestyles
+
+NoteStyles Change How The Notes Look
+
+to create a NoteStyle simply create a file in `mod_base/data/notestyles/notestyleId.json`.
+
+Heres the format:
+
+```jsonc
+{
+  "version": "1.0.0",
+  "name": "Funkin'", // Name in a chart editor
+  "author": "PhantomArcade", // Creator of the NoteStyle
+  "fallback": null,
+  "assets": {
+    "note": {
+      "assetPath": "shared:notes", // path for notes
```

Comment:
Can you tell me what type of path is this. Do you expect a typical FNF modder to know about internal library system?
 
Neither do I!